packages/usa-date-picker/src/index.js

```diff
@@ -1801,8 +1809,18 @@ const handleMouseoverFromDate = (dateEl) => {
   if (hoverDate === currentCalendarDate) return;
 
   const dateToDisplay = parseDateString(hoverDate);
-  const newCalendar = renderCalendar(calendarEl, dateToDisplay);
-  newCalendar.querySelector(CALENDAR_DATE_FOCUSED).focus();
+  const focusedEl = document.activeElement;
+
+  // If the calendar contains a focused element, reapply focus after new calendar renders.
+  // Targeted using aria-label because they are unique and prevents error when
+  // focus is in the table header.
+  if (calendarEl.contains(focusedEl)) {
+    const focusElLabel = focusedEl.getAttribute("aria-label");
```

Comment:
## activeElement vs. `--focused` class

When troubleshooting this issue, I decided I needed a way to identify what was actually `:focused` and what was hovered. It made sense to use `document.activeElement` to select the focused button natively while maintaining the focused class that is used to identify date range caps. 

This seemed like the path of least resistance though we should consider renaming the `--focused` class to `--hovered` or something similar for the sake of brevity